### PR TITLE
Hexagon Style adjustement

### PR DESCRIPTION
**Pull request by MannixMD**
I hope i fixed the image plus i added icon for the navbar since i forgot about it in the first place

### CODE REVIEW

File: styles/Hexagon/template/event/overall_header_head_append.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{% INCLUDECSS '@dmzx_mchat/mchat.css' %}
+{% INCLUDECSS '@dmzx_mchat/mchat_custom.css' %}
```
Comment (kasimi) [inline styles/Hexagon/template/event/overall_header_head_append.html]: Please add an empty line at the end here.

File: styles/Hexagon/theme/mchat_custom.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.mchat-icon:before {
+	background-image: url("./images/message_icons_orange.png") !important;
+}
+.icon-mchat {
+    background-image: url(./images/icon_mchat.png);
+}
```
Comment (kasimi) [inline styles/Hexagon/theme/mchat_custom.css]: Please add an empty line at the end here.

File: styles/Hexagon/theme/mchat_custom.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.mchat-icon:before {
+	background-image: url("./images/message_icons_orange.png") !important;
+}
+.icon-mchat {
+    background-image: url(./images/icon_mchat.png);
```
Comment (kasimi) [inline styles/Hexagon/theme/mchat_custom.css]: Please quote the URL.

**Comment by kasimi** [comment]
There are still some minor things to fix.

No need to create a new PR each time, you can just push to the existing branch of this PR.

**Comment by MannixMD** [comment]
If only i would knew what to fix it would be nice sorry for the troubles I'm doing this for the first time

**Review by kasimi — CHANGES_REQUESTED** [review]

**Comment by kasimi** [comment]
Oh, I forgot to submit my comments. You should see them now. 😀

**Comment by MannixMD** [comment]
Done :)

**Review by kasimi — APPROVED** [review]

**Comment by kasimi** [comment]
Thanks!

**Pull request merged**